api/src/main/java/org/eclipse/microprofile/config/Config.java

```diff
@@ -69,13 +69,11 @@
      *
      * @param propertyName
      *             The configuration propertyName.
-     * @return The resolved property value as a String-Optional, which will bypass converters.
+     * @return The resolved property value as a String, which will bypass any converters.
      *
-     * @throws IllegalArgumentException
-     *             is thrown if the propertyName maps to an object that is not a
-     *             String.
+     * 
      */
-    Optional<String> getString(String propertyName);
+    String getRawString(String propertyName);
```

Comment:
That said I'm unsure what's the use case for this method in the first place? Should it actually be present here if we are concerned about it being used? Raw values can be obtained from config sources so we may leave it out here in the first round and see whether there's actual demand for it?

Btw. the "raw value" isn't necessarily a String, thinking about a JSON data source for instance. I.e. I'm starting to get doubts whether it's a good idea to use String as the type for representing raw values (here, but more importantly on `ConfigSource`).